src/pages/data/us-daily.js

```diff
@@ -38,17 +39,29 @@ const ContentPage = ({ data }) => (
         {data.allCovidUsDaily.edges.map(({ node }) => (
           <tr>
             <td>
-              {DateTime.fromFormat(node.date.toString(), 'yyyyMMdd').toFormat(
-                'dd LLL yyyy ccc',
-              )}
+              <FormatDate
+                date={DateTime.fromFormat(node.date.toString(), 'yyyyMMdd')}
```

Comment:
Is there a reason why this date format is different from the default one?

Reply:
Unsure, checking.